apps/api/src/app/workflows-v2/usecases/validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase.ts

```diff
@@ -220,4 +243,70 @@ export class PrepareAndValidateContentUsecase {
 
     return result;
   }
+
+  private async computeTierIssues(
+    defaultControlValues: Record<string, unknown>,
+    user: UserSessionData,
+    stepType?: StepTypeEnum
+  ): Promise<Record<string, ContentIssue[]>> {
+    const deferDuration =
+      isValidDigestUnit(defaultControlValues.unit) && isNumber(defaultControlValues.amount)
+        ? calculateMilliseconds(defaultControlValues.amount, defaultControlValues.unit)
+        : 0;
+
```

Comment:
don't you want this calculation inside the usecase, i feel you will be doing it again in the worker

Reply:
worker is a more complex place, you are calculating it separately today for delay or digest, i wouldn't touch this part as part of this PR.

Reply:
is it, wouldn't it be better moving the worker logic inside and both be using it?

Reply:
however you feel